contingent/contingent.markdown

```diff
@@ -1138,7 +1140,7 @@ Here is what the `task` decorator boilerplate looks like:
 
 This is an entirely typical Python decorator declaration.
 It can then be applied to a function
-by naming it after a `@` character atop the `def`
+by naming it after an `@` character atop the `def`
```

Comment:
I pronounce `@` "at", so I made this "an", but maybe the Python pronounciation is different and doesn't require "an"?